applications/crossbar/priv/api/swagger.json

```diff
@@ -35262,16 +35262,21 @@
                 },
                 "max_shortdial_correction": {
                     "default": 5,
-                    "description": "stepswitch maximum shortdial correction",
+                    "description": "add up to {MAX} digits from caller ID to destination number",
                     "minimum": 1,
                     "type": "integer"
                 },
                 "min_shortdial_correction": {
                     "default": 2,
-                    "description": "stepswitch minimum shortdial correction",
+                    "description": "add at least {MIN} digits from caller ID to destination number",
                     "minimum": 1,
                     "type": "integer"
                 },
+                "min_shortdial_destination": {
+                    "default": 0,
+                    "description": "do short-dial correction if dialed number's length is equal to this value ONLY when it is greater than 0",
```

Comment:
I still think this should not default to 0. Instead, I think you could put a `minumum:1` constraint. So if `min_shortdial_destination` is unset/null, it is ignored. If set, the value must be greater than 0 already so use it.

Reply:
Yeah, I forgot to remove that, the code is not relying on `0` as the default value anymore, that's a leftover from the previous changes.